src/components/Uptime.vue

```diff
@@ -33,7 +33,13 @@ export default {
             let key = this.monitor.id + "_" + this.type;
 
             if (this.$root.uptimeList[key] !== undefined) {
-                return Math.round(this.$root.uptimeList[key] * 10000) / 100 + "%";
+                let result = Math.round(this.$root.uptimeList[key] * 10000) / 100;
+                // Only perform sanity check on status page. See louislam/uptime-kuma#2628
+                if (this.$route.path.startsWith("/status")) {
```

Comment:
`result + "%"` is duplicated, it could be simplified by merging into one condition.

```
if (this.$route.path.startsWith("/status") && result > 100) {
```

Reply:
Reducing code is not always about simplifying it

Reply:
I have gone and updated this